MultiVolumeImporterPlugin.py

```diff
@@ -78,16 +78,17 @@ def examine(self,fileLists):
     allfiles = []
     for files in fileLists:
       loadables += self.examineFiles(files)
+
+      # this strategy sorts the files into groups
+      loadables += self.examineFilesIPPAcqTime(files)
+
       allfiles += files
 
     # here all files are lumped into one list for the situations when
     # individual frames should be parsed from series
     loadables += self.examineFilesMultiseries(allfiles)
 
     # this strategy sorts the files into groups
-    loadables += self.examineFilesIPPAcqTime(allfiles)

```

Comment:
Please keep this call, as it is needed to handle Dynamic Susceptibility Contrast (DSC) datasets produced by some vendors (either Siemens, or GE, or both...).